dts/riscv/espressif/esp32c3/esp32c3_common.dtsi

```diff
@@ -71,6 +71,7 @@
 
 	esp32_bt_hci: esp32_bt_hci {
 		compatible = "espressif,esp32-bt-hci";
+		bt-hci-vs-ext;
```

Comment:
Is this really correct, i.e. the ESP32 controller supports the Zephyr HCI vendor extensions? It's great if it does - I'm just surprised, since I thought the Zephyr native (open source) controller is the only one that has implemented those, and everyone else does vendor extensions based on the vendor of the controller.

Reply:
@jhedberg Most BLE vendors have command extensions. It is usually used at the driver level so Zephyr stack doesn't (need to) know about them.

Reply:
Sure, but this property is specifically about the *Zephyr* VS extensions. It shouldn't be used to indicate support for any other vendor command/event set. If we ever wanted to make this more generic, what could make sense is to use a 16 bit value to indicate the specific company id whose extensions are available, but that's not what we have today.

Reply:
And to be even more clear: when this property is set the Zephyr host will assume the availability of Zephyr VS extensions and try to use them. So it's really not just something only for drivers.